client/js/middleware/serialization/serializer_utils.js

```diff
@@ -0,0 +1,8 @@
+import _    from 'lodash';
+
+export function scrub(item) {
+  let scrubbedItem = _.cloneDeep(item);
+  scrubbedItem = _.omitBy(scrubbedItem, _.isNil);
+  scrubbedItem = _.omitBy(scrubbedItem, _.isEmpty);
```

Comment:
`_.isEmpty(11)` returns true, it is in the docs and I tried it. I think it should be easy to avoid though if you just check to see if something is a string, or a number before calling `isEmpty` on it